discovery_engine_core/core/src/stack/ops/breaking.rs

```diff
@@ -120,8 +118,7 @@ impl Ops for BreakingNews {
 
     fn merge(&self, stack: &[Document], new: &[Document]) -> Result<Vec<Document>, GenericError> {
         let merged: Vec<Document> = chain!(stack, new).cloned().collect();
-        let filtered = filter_semantically(merged, &self.semantic_filter_config);
-        Ok(filtered)
+        Ok(merged)
```

Comment:
:art: since this is essentially a one liner now, we could write it as
```suggestion
        Ok(chain!(stack, new).cloned().collect())
```